localstack-docker/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM localstack/localstack
+FROM localstack/localstack:0.12.20
```

Comment:
0.13.0 doesnt load the api_recorded_calls.json and so doesn't create the bucket. Need to find how to fix it.